go.mod

```diff
@@ -5,26 +5,28 @@ go 1.14
 require (
 	github.com/buildpacks/pack v0.18.0
 	github.com/containers/image/v5 v5.10.5
-	github.com/docker/docker v20.10.0-beta1.0.20201110211921-af34b94a78a1+incompatible
+	github.com/docker/docker v20.10.2+incompatible
 	github.com/docker/go-connections v0.4.0
 	github.com/markbates/pkger v0.17.1
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/ory/viper v1.7.4
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.1.3
-	golang.org/x/crypto v0.0.0-20201016220609-9e8e0b390897
+	golang.org/x/term v0.0.0-20201210144234-2321bbc49cbf
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.18.12
-	k8s.io/apimachinery v0.18.12
-	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
-	knative.dev/client v0.20.0
-	knative.dev/eventing v0.20.0
-	knative.dev/pkg v0.0.0-20210107022335-51c72e24c179
-	knative.dev/serving v0.20.0
+	k8s.io/api v0.19.7
+	k8s.io/apimachinery v0.19.7
+	k8s.io/client-go v0.19.7
+	knative.dev/client v0.21.0
+	knative.dev/eventing v0.21.0
+	knative.dev/pkg v0.0.0-20210216013737-584933f8280b
+	knative.dev/serving v0.21.0
 )
 
 replace (
+	// knative.dev/serving@v0.21.0 and knative.dev/pkg@v0.0.0-20210331065221-952fdd90dbb0 require different versions of go-openapi/spec
+	github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.6
 	// Nail down k8 deps to align with transisitive deps
-	k8s.io/client-go => k8s.io/client-go v0.18.12
-	k8s.io/code-generator => k8s.io/code-generator v0.18.12
+	k8s.io/client-go => k8s.io/client-go v0.19.7
+	k8s.io/code-generator => k8s.io/code-generator v0.19.7
```

Comment:
Those 2 lines should be OK to remove, as Knative 0.21 should be aligned k8s client version and don't pull any unexpected ones.